### PR TITLE
bluetooth: mesh: doc: Add missing Doxygen comments

### PR DESCRIPTION
**Pull request by kartben**
Misc. improvements in the Doxygen comments

### CODE REVIEW

File: include/zephyr/bluetooth/mesh/access.h [hunk 1]
```diff
@@ -41,14 +41,21 @@
...
+#define BT_MESH_ADDR_FRIENDS       0xfffd   /**< all-friends */
+#define BT_MESH_ADDR_PROXIES       0xfffc   /**< all-proxies */
+#define BT_MESH_ADDR_DFW_NODES     0xfffb   /**< all-directed-forwarding-nodes */
+#define BT_MESH_ADDR_IP_NODES      0xfffa   /**< all-ipt-nodes */
+#define BT_MESH_ADDR_IP_BR_ROUTERS 0xfff9   /**< all-ipt-border-routers */
+/**
+ * @}
+ */
 
 #define BT_MESH_KEY_UNUSED        0xffff
```
Comment (PavelVPV) [inline include/zephyr/bluetooth/mesh/access.h:60]: Could you please also add description for macros at lines 60 - 73?

**Review by PavelVPV — COMMENTED** [review]
Thank you for fixing this! One small comment.

File: include/zephyr/bluetooth/mesh/access.h (hunk 1 above)
Comment (PavelVPV) [reply include/zephyr/bluetooth/mesh/access.h:60]: I couldn't come up with any better than this:
```
/**
 * @name Predefined key indexes
 * @{
 */
#define BT_MESH_KEY_UNUSED        0xffff          /**< Key unused */
#define BT_MESH_KEY_ANY           0xffff          /**< Any key index */
#define BT_MESH_KEY_DEV           0xfffe          /**< Device key */
#define BT_MESH_KEY_DEV_LOCAL     BT_MESH_KEY_DEV /**< Local device key */
#define BT_MESH_KEY_DEV_REMOTE    0xfffd          /**< Remote device key */
#define BT_MESH_KEY_DEV_ANY       0xfffc          /**< Any device key */
/**
 * @}
 */
```

Comment (kartben) [reply include/zephyr/bluetooth/mesh/access.h]: added!

**Review by PavelVPV — APPROVED** [review]

**Review by alxelax — APPROVED** [review]

**Review by Andrewpini — APPROVED** [review]

**Pull request merged**